bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -563,7 +563,21 @@ public void sendCommand(String io, String command, String params, String url) {
 
         Boolean result = sendCommandInternal(io, command, params, url);
         if (!result) {
-            sendCommandInternal(io, command, params, url);
+            scheduler.schedule(() -> {
+                repeatSendCommandInternal(io, command, params, url, thingConfig.getRetries());
+            }, thingConfig.getRetryDelay(), TimeUnit.MILLISECONDS);
+        }
+    }
+
+    private void repeatSendCommandInternal(String io, String command, String params, String url, int retries) {
+        logger.info("Retrying command, retries left: {}", retries);
+        Boolean result = sendCommandInternal(io, command, params, url);
+        if (result != null && !result) {
+            if (retries > 0) {
```

Comment:
result cannot be null at this place. Please use only one if with two conditions

Reply:
Done and changed type to boolean